nyc_data/ppe/data_mappings.py

```diff
@@ -145,7 +145,6 @@ def to_objects(self):
             quantity=self.quantity,
             vendor=self.vendor,
             raw_data=self.raw_data,
-            data_source=DataType.EDC_PPE,
```

Comment:
These `data_source` properties were never used, correct? Since there wasn't a `data_source` column in the Purchase,  Delivery, etc. tables

Reply:
They were used in the old provenance system -- we marked everything from the data source we were loading as replaced=true during the import to avoid having duplicate data.

In the new system, this is captured with the foreign key to the dataimport row